internal/services/global_params.go

```diff
@@ -28,18 +30,24 @@ func (s *Service) fetchAndSaveParams(ctx context.Context) error {
 		return fmt.Errorf("failed to save checkpoint params: %w", err)
 	}
 
-	allStakingParams, err := s.bbn.GetAllStakingParams(ctx)
+	allStakingParams, err := s.bbn.GetStakingParams(ctx, s.stakingParamsLatestVersion)
```

Comment:
reminder: we need to use next version (but keep in mind that first value must to be zero)